### ISSUE TITLE
[Accessibility] It is impossible to start Insignificant Little Vermin with the keyboard

### ISSUE THREAD

**Issue by citelao** [filiph/edgehead#7]
# Repro #

I cannot start Insignificant Little Vermin with the keyboard!

The green "START" button does not accept keyboard focus, so I cannot tab to it. When I press tab after loading `https://egamebook.com/vermin/v/latest/`, my first move lands on the "Start" link, which seems to do nothing, and my second move disappears.

In reality, focus is going to the buttons hidden behind the start screen (this can be verified using a screen reader: open the screen reader, press tab, and hear it read "Reload, button" and "Info, button", etc). Further item navigation lets me play the game, without dismissing the Start screen!

If I navigate to the button using a screenreader, then activate it manually, I can dismiss the start screen. But that behavior is hard to discover, especially when tabbing can get me to the game!

# Expected behavior #

I should only be able to tab to visible items. I should not be able to start the game with the start screen still visible.

# Potential mitigations #

There are a couple changes here:

* The START button should be a `<button>`. If it *absolutely cannot* be, it [should have](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role) `role="button"` and `tabindex="0"`
* It should not be possible to access the game from behind the start screen 😁. Potential fixes:
    * Create a keyboard trap to wrap tab and shift-tab while the screen is open (see step 5 of [this guide on accessible modal dialogs](https://bitsofco.de/accessible-modal-dialog/))
    * **Suggested:** Make the start screen truly a new page
    * Bodge: dismiss the start screen on the first button press in the game, so sighted users can at least follow along with their screenreading friends. Some screenreader users are sighted, though, so this could be very confusing 😬.
* Make the first "Start" link do something besides reload the page. Why is it a link in the first place?

Let me know if I can clarify anything.
